### PR TITLE
Add quick start example

### PR DESCRIPTION
**Pull request by ifavo**
This PR aims to simplify the start for impatient developers on the app template:

- Adds a `QuickStart.md` with reduced information and a single snippet to get started
- Documents the links that will be available
- Defaults the log dir, to use less config variables

### CODE REVIEW

**Review by Agilulfo1820 — APPROVED** [review]

File: QuickStart.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+yarn contracts:solo-up
+yarn contracts:deploy:solo
+
+PORT=3000 \
+ ORIGIN="*" \
+ OPENAI_API_KEY="sk-proj-.." \
+ ADMIN_MNEMONIC="denial kitchen pet squirrel other broom bar gas better priority spoil cross" \
+ NETWORK_URL=http://localhost:8669 \
+ REWARD_AMOUNT=1 \
+ yarn dev
```
Comment (pierobassa) [inline QuickStart.md:23]: Like the idea of the quickstart! But 

`OPENAI_API_KEY="sk-proj-.."` would configure the backend with a wrong API key

Users might mistake the steps as a copy and paste and everything will work.

Possible solution could be adding the "Result" section of the Quickstart below the Readme here:
image

**Review by pierobassa — CHANGES_REQUESTED** [review]
Adding Request Change

**Comment by pierobassa** [comment]
> > Possible solution could be adding the "Result" section of the Quickstart below the Readme here:
> 
> 👋 not sure what you mean by that. Would this be adding the link table to the first section?
> 
> I agree with the potential copy & paste misunderstanding. It is stated at the top as required variable.
> 
> I went back with my original frustration about the required amount of ENV and found that `envalid` is used to verify `PORT` and `NODE_ENV` only. I've modified it slightly to verify all the ENV and also use the verified ENV. In the process adjusted, it now also sets some default values **and** has an OpenAPI-Key verifier, that informs the user where to obtain one.
> 
> What do you think about the change?

Nice catch! Could you have a reference to the `Quickstart.md `in the `Readme.md` too?

**Comment by ifavo** [comment]
I did add a link to the README, hope its what you expect. Feel free to adjust the way you like to see it!

**Review by pierobassa — APPROVED** [review]

**Pull request merged**